repo/packages/U/ubuntu/0/config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "type": "object",
+    "properties": {
+      "service": {
+        "description": "Configuration properties for the ubuntu service for DC/OS.",
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The name of the service to display in the DC/OS dashboard.",
+            "type": "string",
+            "default": "ubuntu"
+          },
+          "cpus": {
+            "description": "CPU shares to allocate to this ubuntu instance.",
+            "type": "number",
+            "default": 0.5,
+            "minimum": 0.1
+          },
+          "mem": {
+            "description": "Memory (in MB) to allocate to this ubuntu instance.",
+            "type": "number",
+            "default": 2048,
+            "minimum": 256
+          }
+        },
+        "required": ["name", "cpus", "mem"]
+      },
+      "ubuntu": {
+        "description": "ubuntu specific configuration",
+        "type": "object",
+        "properties": {
+          "password": {
+            "type": "string",
+            "description": "Password",
+            "default": "changeme"
+          }
+        }
+      },
+      "networking": {
```

Comment:
this category should probably use `x-form` to hide as we don't want user to interact with them